Homework1/task5/task5.py

```diff
@@ -0,0 +1,13 @@
+from typing import List
+
+
+def find_maximal_subarray_sum(nums: List[int], k: int) -> int:
+    if len(nums) < k or k == 0:
+        return "Error,pls write correct list"
```

Comment:
if k==0 return 0
but case when k is bigger than nums length if perfectly fine, u dont need additional handling for this